### PR TITLE
Added mention option for /repeat

### LINKED ISSUE #82: repeat command should be able to 'at' a user
The repeat command should have the ability to 'at' a designated user, not just the person who called it
**Existing**
image

**Proposed**
`/repeat channels @someone`

should produce:

`@someone: check out the channel guide (sent by: @sender)`

### PR DESCRIPTION
**Pull request by ptrstr**
Converted the `text` variable to an `argument` variable.

Resolves #82

### CODE REVIEW

File: pybot/endpoints/slack/commands.py [hunk 1]
```diff
@@ -112,9 +112,10 @@ async def slash_repeat(command: Command, app: SirBot):
     channel_id = command["channel_id"]
     slack_id = command["user_id"]
     slack = app["plugins"]["slack"].api
+    arguments = command["text"].split(" ")
 
     method_type, message = get_slash_repeat_messages(
-        slack_id, channel_id, command["text"]
+        slack_id, channel_id, arguments
```
Comment (joaramirezra) [inline pybot/endpoints/slack/commands.py:118]: where is arguments defined ?

File: pybot/endpoints/slack/utils/slash_repeat.py [hunk 2]
```diff
@@ -36,16 +36,13 @@ def modify_params(modify_options: dict) -> dict:
...
-    ] = f'<@{modify_options["slack_id"]}>: {modify_options["pretext"]}'
-    message["attachments"][0]["title"] = modify_options["title"]
+    message["attachments"][0]["pretext"] = f'<{modify_options["arguments"][1]}>: {modify_options["pretext"]} (sent by: <@{modify_options["slack_id"]}>)' if len(modify_options["arguments"]) >= 2 else f'<@{modify_options["slack_id"]}>: {modify_options["pretext"]}'
     message["attachments"][0]["title_link"] = modify_options["link"]
 
     return message
 
 
-def repeat_items(requested_text: str, slack_id: str, channel_id: str) -> dict:
+def repeat_items(arguments: list, slack_id: str, channel_id: str) -> dict:
```
Comment (joaramirezra) [inline pybot/endpoints/slack/utils/slash_repeat.py:45]: i see it my bad

**Comment by aaron-suarez** [comment]
@ptrstr I'm about to review this PR. In the future, please create a new branch rather than working directly on staging when you open a PR. Thanks

**Comment by aaron-suarez** [comment]
It appears to work
Screen Shot 2020-10-29 at 1 33 22 PM

I'll go ahead and merge and see if staging has a solution for that `text of link` to actually say `Check out the channel guide!` like it's supposed to.

Thanks for this!

**Review by aaron-suarez — APPROVED** [review]

**Pull request merged**